### PR TITLE
feat: add badges to the README

### PR DESCRIPTION
**Pull request by rachwalk**
This PR adds badges for the README, plus commented badges, which will work once the repo is made public, CI setup, and release is made.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # 🦊 RAI
 
+![Static Badge](https://img.shields.io/badge/Python-3.10-blue)
+![Static Badge](https://img.shields.io/badge/ROS2-humble-blue)
```
Comment (adamdbrw) [inline README.md]: we aim for jazzy primary. Right now, we are testing on jazzy, right? We will support humble, iron and jazzy.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,12 @@
 # 🦊 RAI
+![Static Badge](https://img.shields.io/badge/Python-3.10-blue)
+![Static Badge](https://img.shields.io/badge/ROS2-humble-blue)
```
Comment (adamdbrw) [inline README.md]:
```suggestion
![Static Badge](https://img.shields.io/badge/ROS2-jazzy-blue)
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # 🦊 RAI
 
+![Static Badge](https://img.shields.io/badge/Python-3.10-blue)
```
Comment (adamdbrw) [inline README.md]: Could we integrate with our CI and add a badge?

File: README.md [hunk 4]
```diff
@@ -1,5 +1,13 @@
 # 🦊 RAI
```
Comment (adamdbrw) [inline README.md:2]:
```suggestion
![Static Badge](https://img.shields.io/badge/Ubuntu-24.04-orange)
```

Comment (adamdbrw) [reply README.md:2]: Or check if there's a standard one

File: README.md (hunk 1 above)
Comment (rachwalk) [reply README.md]: I believe that the general practice is to list the lowest supported version, but in general we aim for Python 3.12 and Jazzy, yes. Should the python version be also set to 3.12 in this case?

File: README.md (hunk 3 above)
Comment (rachwalk) [reply README.md]: This line:
```
[comment]: # ![GitHub Actions Workflow Status](https://img.shields.io/github/actions/workflow/status/RobotecAI/rai/poetry-test.yml)
```

Once uncommented will show the CI status badge on the rai repository. I can uncomment it, but given that the public rai repository currently doesn't have CI integrated will show an error. There is no easy way to show CI status for a private repository (requires self hosted shields.io server). The same goes for the currently commented line with Release version - public rai repo doesn't have a release to be shown.

File: README.md (hunk 1 above)
Comment (maciejmajek) [reply README.md]: Yes, please. 
Our default stack is: Ubuntu 24.04, ROS 2 Jazzy and python 3.12

Comment (rachwalk) [reply README.md]: fixed in 9bc5ac214be7633e0330ae068cec556a0d668c64

**Review by adamdbrw — APPROVED** [review]

**Pull request merged**